Doc/c-api/refcounting.rst

```diff
@@ -8,7 +8,8 @@ Reference Counting
 ******************
 
 The macros in this section are used for managing reference counts of Python
-objects.
+objects.  Note that :c:macro:`Py_REFCNT` and :c:macro:`Py_SET_REFCNT` are
+documented as a part of :c:type:`PyObject`.
```

Comment:
IMHO, I think it would be cleaner to move those from `structures.rst` to here, rather than just referring to them.  They are clearly in the "reference counting" category.

Reply:
you know, i started out assuming there was a reason they were elsewhere, but now that you mention it this is just another area describing functions that apply specifically to PyObjects, so there is really no justification i can think of for why Py_REFCNT is any different

ima push in a bit

Reply:
> IMHO, I think it would be cleaner to move those from `structures.rst` to here, rather than just referring to them. They are clearly in the "reference counting" category.

hey anybody you know of I could bug to get this merged?

Reply:
> IMHO, I think it would be cleaner to move those from `structures.rst` to here, rather than just referring to them. They are clearly in the "reference counting" category.

hey any idea who to bug to try to get this merged?

Reply:
I think the reason they were in structures.rst is that these macros directly get/set a field on the PyObject, just like the macros listed above and below it. I agree that these macros are best placed in the refcounting docs, but we should also keep a reference to them in structure.rst.

Also, there's a merge conflict now. Sorry for the wait; once these issues are resolved I'm happy to merge this PR.

Reply:
> I think the reason they were in structures.rst is that these macros directly get/set a field on the PyObject, just like the macros listed above and below it. I agree that these macros are best placed in the refcounting docs, but we should also keep a reference to them in structure.rst.
> 
> Also, there's a merge conflict now. Sorry for the wait; once these issues are resolved I'm happy to merge this PR.

how about that?